### DISCUSSION TITLE
Where is the help text?

### DISCUSSION THREAD

**Discussion by mogul** [Lurkars/gloomhavensecretariat#559] [Q&A]
I want to supply fixes to the help text at https://help.gloomhaven-secretariat.de/ but I can't figure out where the source is. It is up for PRs somewhere I'm not seeing?

**Comment by Lurkars** [comment]
Good point, help site is not on GitHub, but I will add it and then you can create PR!

**Comment by Lurkars** [comment]
Here you go https://github.com/Lurkars/ghs-help

**Reply by mogul** [reply]
Terrific, thank you!

**Comment by mogul** [comment]
You have a [PR pending](https://github.com/Lurkars/ghs-help/pull/1). Let me take this opportunity to once again thank you for all your hard work, which we marvel over every Friday night!
